src/ansys/tools/path/path.py

```diff
@@ -1156,7 +1156,11 @@ def version_from_path(product: PRODUCT_TYPE, path: str) -> int:
 
     """
     if not isinstance(path, str):
-        raise ValueError(f"Provided path '{path}' is not a string.")
+        raise ValueError(
+            f'The provided path, "{path}", is not a valid string.'
+            f'If "{product}" is not installed in the default location, use ``save-ansys-path``'
+            f"to save the path so it can be found by ``ansys-tools-path``."
```

Comment:
```suggestion
            f'The provided path, "{path}", is not a valid string. '
            f'If "{product}" is not installed in the default location, use ``save-ansys-path`` '
            f"to save the path so it can be found by ``ansys-tools-path``."
```